Sources_v3/StreamChatUI/ChatChannel/Attachments/ChatMessageAttachmentsView.swift

```diff
@@ -121,8 +141,10 @@ open class ChatMessageAttachmentsView<ExtraData: ExtraDataTypes>: View, UIConfig
         for attachment in content?.attachments ?? [] {
             if !attachment.actions.isEmpty {
                 options.insert(.interactiveAttachment)
-            } else if attachment.isImageOrGIF {
```

Comment:
Do we any place where `isImageOrGIF` is used? Looks it's not relevant anymore

Reply:
Yes, it's deleted in 2c7a17b